velox/dwio/common/SelectiveColumnReader.cpp

```diff
@@ -220,6 +220,54 @@ void SelectiveColumnReader::getIntValues(
   }
 }
 
+void SelectiveColumnReader::getUnsignedIntValues(
+    RowSet rows,
+    const TypePtr& requestedType,
+    VectorPtr* result) {
+  switch (requestedType->kind()) {
+    case TypeKind::SMALLINT:
+      switch (valueSize_) {
+        case 4:
+          getFlatValues<int32_t, int16_t>(rows, result, requestedType);
+          break;
+        default:
+          VELOX_FAIL("Unsupported value size: {}", valueSize_);
+      }
+      break;
+    case TypeKind::INTEGER:
+      switch (valueSize_) {
+        case 4:
+          getFlatValues<int32_t, int32_t>(rows, result, requestedType);
```

Comment:
Should it be `getFlatValues<uint32_t, int32_t>`? Even though it does not make a difference, using uint32_t here will make the code easy to understand.

Reply:
yes, here `uint32_t` doesn't make difference, but it make code better understand. So updated.